requirements.txt

```diff
@@ -1,18 +0,0 @@
-#
```

Comment:
It looks like the switch from `requirements.txt` to `requirements.prod.txt` has also resulted in a version upgrade. Job-runner's deployment is a bit weird because we can't use Docker (because Windows) and neither can we pip install stuff (because we're in the secure networkless environment). So we've ended up shipping in dependencies via this repo:
https://github.com/opensafely-core/job-runner-dependencies

This means that upgrading stuff isn't completely straightforward and so it would be best to keep to the currently specified versions so we don't risk drift between dev/CI and prod.

Hopefully we can revisit this whole situation and do something better once we're running on a Linux VM.

Reply:
This definitely wasn't intended, thanks for catching it!

Reply:
Updated by copying over the versions from requirements.txt -> requirements.prod.txt.  With the way just is configured we're going to see some changes creep into requirements.prod.txt, for instance `ruamel-yaml-clib` is picked up from `ruamel-yaml` (both have the `.` normalised to `-`).  Is there a reason we don't include that already?

Reply:
I think we can leave this as-is and see how we get on with it then?

Reply:
Yep, agreed

Reply:
@evansd – could you add the green tick please 🙂